modules/sdk-coin-eth/src/eth.ts

```diff
@@ -2096,15 +2096,16 @@ export class Eth extends BaseCoin {
    * @param typedData the typed data to prepare
    * @return a buffer of the result
    */
-  encodeTypedData(typedData: TypedData<never>): Buffer {
+  encodeTypedData(typedData: TypedData): Buffer {
```

Comment:
Changes to a required parameters on a public method is a breaking change. Needs to be documented in its' own commit for `sdk-coin-eth` separate from other modules.